models/majorModel.js

```diff
@@ -1,8 +1,50 @@
-var mongoose = require('mongoose');
+const mongoose = require('mongoose');
 
-// Create major Schema
-var majorSchema = mongoose.Schema({
-  sample: String,
+const majorSchema = mongoose.Schema({
+  name: {
+    type: String,
+    // This default is set mainly to show off what the major is
+    // intended to look like
+    default: 'Mechanical Engineering',
+  },
+  generalRequirements: {
+    type: [String],
+    // Because these are the same for every student, it makes sense to
+    // include default general requirements regardless of major
+    default: [
+      'Modelling and Simulation of the Physical World',
+      'Linearity I',
+      'Linearity II',
+      'Probability and Statistics',
+      'Biology Foundation',
+      'Chemistry/Materials Science',
+      'Physics',
+      'Introduction to Sensors, Instrumentation, and Measurement',
```

Comment:
I think the cleanest way to do this might be to have two separate requirements, one for ModCon and one for RWM. Then ISIM can just fulfill both of those requirements.